frontend/typings/restangular/restangular.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for Restangular v1.4.0
```

Comment:
The restangular version in the bower.json is 1.5.1. So either only this comment is outdated or the type definitions have to be updated as well.